data/json/proficiencies/misc.json

```diff
@@ -7,6 +7,22 @@
     "can_learn": true,
     "time_to_learn": "10 h"
   },
+  {
+    "type": "proficiency",
+    "id": "prof_basketweaving",
+    "name": { "str": "Basketweaving" },
+    "description": "Forming coarse fibers into shapes.",
+    "can_learn": true,
+    "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_carving",
+    "name": { "str": "Woodcarving" },
+    "description": "Forming coarse fibers into shapes.",
```

Comment:
Is this supposed to be the same as the Basketweaving description?

Reply:
Hmm.

Reply:
It's technically correct.